service_container/configurators.rst

```diff
@@ -186,6 +186,23 @@ You can configure the service configurator using the ``configurator`` option:
             ->setConfigurator(array(new Reference('app.email_configurator'), 'configure'))
         ;
 
+.. note::
+
+    The traditional configurator syntax in YAML files used an array to define
+    the service id and the method name:
+
+    .. code-block:: yaml
+
+        app.newsletter_manager:
+            # new syntax
+            configurator: 'app.email_configurator:configure'
+            # old syntax
+            configurator: ['@app.email_configurator', configure]
+
+    .. versionadded:: 3.2
```

Comment:
The `versionadded` directive nested in the `note` block doesn't look very nice. What about instead transforming the `note` into a `versionadded` instead as @stof suggested.